src/storage/secondary/block/rle_block_builder.rs

```diff
@@ -2,34 +2,109 @@
 
 use std::borrow::Borrow;
 
-use bytes::BufMut;
+use bytes::{Buf, BufMut};
+use prost::DecodeError;
 use risinglight_proto::rowset::BlockStatistics;
 
 use super::BlockBuilder;
 use crate::array::Array;
 
+pub fn encode_32<B>(mut value: u32, buf: &mut B)
```

Comment:
Would you please write docstring for this function? Also, would you please explain a little bit about the implementation? (e.g. borrowed from other products? Or some general ideas about this)

Reply:
Yeah, In fact, I borrow the idea from the prost::encode_varint.
This function encodes u32 to multiple u8 , and for every u8, the highest bit tells whether this u8 belongs to the same u32.

Reply:
Please add license header of prost in this file.